cla-backend-go/v2/dynamo_events/service.go

```diff
@@ -128,8 +128,9 @@ func NewService(stage string,
 	s.registerCallback(githubOrgTableName, Modify, s.GitHubOrgUpdatedEvent)
 	s.registerCallback(githubOrgTableName, Remove, s.GitHubOrgDeletedEvent)
 
-	s.registerCallback(repositoryTableName, Insert, s.GithubRepoAddedEvent)
-	s.registerCallback(repositoryTableName, Remove, s.GithubRepoDeletedEvent)
+	s.registerCallback(repositoryTableName, Insert, s.GithubRepoModifyAddEvent)
+	s.registerCallback(repositoryTableName, Modify, s.GithubRepoModifyAddEvent)
+	s.registerCallback(repositoryTableName, Remove, s.GithubRepoModifyAddEvent)
```

Comment:
If we are registering for the remove event, we need to check for the event type and if Remove, we need to unmarshall the `oldImage`, not the `newImage`.